cli/replicasets/membership.go

```diff
@@ -0,0 +1,134 @@
+package replicasets
```

Comment:
Membership is an implementation detail of cartridge. You don't have any guarantees it will work in future or it won't be changed to built-in "swim" module. I'm strictly against this module, I'm strictly against usage of membership. Please, use only cartridge API.

Reply:
Please, show me the way to do this things without using a membership and I will rewrite this code.

Reply:
Well, it was discussed verbally. Ok.